src/view.rs

```diff
@@ -78,6 +72,20 @@ pub fn handle_event<'a>(event: &Event, state: &'a State) -> Option<SpacerunEvent
     None
 }
 
+fn select_command<'a>(command: &'a Command) -> Option<SpacerunEvent<'a>> {
+    match command {
+        command @ Command::Node(_) => return Some(SpacerunEvent::SelectCommand(command)),
+        Command::Leaf(child_leaf) => {
```

Comment:
vielleicht könnte das auch in `handle_event` passieren?

Reply:
Hmm, Ich wollte die Logik son bils extrahieren da `handle_event` langsam etwas groß wird

Reply:
@flodiebold Gibt es einen Grund, Logik nicht in er Art auszulagern?

Reply:
oh sorry, ich meinte eigentlich das match weiter oben wo auch `SelectCommand` behandelt wird... und den cli-aufruf könnte man natürlich trotzdem auch noch in ne funktion auslagern, aber ich finds halt komisch dass wenn ein prozess ausgeführt wird das hier passiert, aber wenn ein command ausgewählt wird wird hier nur `SelectCommand` zurückgegeben und das weiter oben behandelt